lib/ecto_job/job_queue.ex

```diff
@@ -309,12 +313,15 @@ defmodule EctoJob.JobQueue do
 
   @doc """
   Computes the expiry time for an `"IN_PROGRESS"` job based on the current time and attempt counter
+
+  Note that the integer value represents seconds to conform with unix timestamp values,
+  not milliseconds
   """
   @spec progress_expiry(DateTime.t(), integer) :: DateTime.t()
-  def progress_expiry(now, attempt) do
+  def progress_expiry(now, attempt, base_expiry_seconds \\ 300) do
```

Comment:
Since the 300s value is defaulted in `Config`, I think it is fine to remove the `\\ 300` here.

Reply:
Sure thing. I only added the default because this is a public function and it would be a breaking change if someone was calling it in the application code.